charts/kyverno-policies/templates/pod-sec-ctxt-customer-access-ingress.yaml

```diff
@@ -0,0 +1,42 @@
+{{- $name := "pod-sec-ctxt-customer-access-ingress" }}
```

Comment:
If this is an istio ingress gateway, remove. Same for similar workloads. Remove the validation policies as well.

Reply:
We have already filtered out the istio-gateway related resources. This has nothing to do with it!

Reply:
okay.